addons/mail/static/src/core_ui/thread.js

```diff
@@ -33,6 +34,7 @@ export class Thread extends Component {
     static props = [
         "isInChatWindow?",
         "hasScrollAdjust?",
+        "jumpPresent?",
```

Comment:
In Discuss code yes it's always used, because we have a composer all the time. But I think it's good practice to reduce the amount of mandatory props to the minimum, so components are easier to reuse.

Here it makes sense to not make handling of `jumpPresent` mandatory. It's not a programming error, just a convenience for reusing `<Thread>`